packages/@ember/-internals/glimmer/tests/integration/components/input-angle-test.js

```diff
@@ -582,17 +577,12 @@ moduleFor(
     ['@test sends an action with `<Input @enter={{action "foo"}} />` when <enter> is pressed'](
       assert
     ) {
-      assert.expect(2);
+      assert.expect(1);
 
       this.render(`<Input @enter={{action 'foo'}} />`, {
         actions: {
-          foo(value, event) {
+          foo() {
             assert.ok(true, 'action was triggered');
-            if (jQueryDisabled) {
-              assert.notOk(event.originalEvent, 'event is not a jQuery.Event');
-            } else {
-              assert.ok(event instanceof jQuery.Event, 'jQuery event was passed');
-            }
```

Comment:
Can we assert the presence of `event` here and `event instanceof Event` or something?

I believe these two comments apply across the rest of this file and some other files. In the test which isn't for deprecated behavior, please assert the `event` argument is passed.

Reply:
Not only does this totally make sense, but trying to add that assertion uncovered an actual bug, as the event passed *was* a jQuery event. 😱

The internal components seem do use some additional jQuery related magic, that I was not aware of: https://github.com/emberjs/ember.js/blob/3ce13cea235cde8a87d89473533c453523412764/packages/%40ember/-internals/glimmer/lib/components/internal.ts#L556-L576

So this will need some more cleanup...